Reference/google/protobuf/unittest_custom_options.pb.swift

```diff
@@ -1367,36 +1539,66 @@ public struct ProtobufUnittest_Aggregate: ProtobufGeneratedMessage {
 
   struct Extensions {
 
-    static let Google_Protobuf_FileOptions_nested = ProtobufGenericMessageExtension<ProtobufOptionalMessageField<ProtobufUnittest_Aggregate>, Google_Protobuf_FileOptions>(protoFieldNumber: 15476903, protoFieldName: "nested", jsonFieldName: "nested", swiftFieldName: "ProtobufUnittest_Aggregate_nested", defaultValue: nil)
+    static let Google_Protobuf_FileOptions_nested = ProtobufGenericMessageExtension<ProtobufOptionalMessageField<ProtobufUnittest_Aggregate>, Google_Protobuf_FileOptions>(protoFieldNumber: 15476903, protoFieldName: "nested", jsonFieldName: "nested", swiftFieldName: "ProtobufUnittest_Aggregate_nested", defaultValue: ProtobufUnittest_Aggregate())
```

Comment:
I'm not sure I follow you here: being proto2, extensions can have explicit default values as well ( see https://github.com/google/protobuf/blob/master/src/google/protobuf/unittest_custom_options.proto#L65-L70). How are they implied by the types?